web/models/stripe_account.ex

```diff
@@ -0,0 +1,38 @@
+defmodule CodeCorps.StripeAccount do
+  @moduledoc """
+  Represents a StripeAccount stored on Code Corps.
+  """
+
+  use CodeCorps.Web, :model
+
+  @primary_key {:id, :string, []} # set primary key as string
+  @derive {Phoenix.Param, key: :id} # set method of determining primary key for route
+  schema "stripe_account" do
+    field :business_name, :string
```

Comment:
I'm curious if we want to store the `business_name` on the StripeAccount since each StripeAccount belongs to an organization, where I presume the business name and organization name are the same. But perhaps having the business name defined on the stripe account meets a feature requirement? :)